### ISSUE TITLE
SBT 1.4.7 no longer generates Junit style xml reports

### ISSUE THREAD

**Issue by anthonyemerson** [sbt/sbt#6324]
sbt version: 1.4.7 (Last know to be working in sbt 1.2.8 but that is just the last version I was using before this upgrade)
scala version: 2.13.4
scalatest version: 3.2.2 (Utilizing FunSpec)

Steps
Assuming an existing test framework:
Run a spec in a Jenkins job utilizing the `sbt testOnly` command

Problems
The Junit style xml files that were previously automatically generated by sbt are no longer present. The html files are there, but Jenkins utilizes the xml files to generate its test reports and to set the build status of a job.

Expectations
In the target folder I expect there to be the xml files that Jenkins needs to report on the results of my test runs

Notes (ie the long version)
I am trying to update from sbt 1.2.8 up to 1.4.7 but when I try to run my tests in Jenkins all the Junit style xml files are missing. The docs still say the Junit Plugin is on by default, but just in case I tried setting `val myProject = (project in file(".")).enablePlugins(plugins.JUnitXmlReportPlugin)` as the docs suggest using `disablePlugins` to turn it off I just assumed the opposite would work. Unfortunately it didn't, nothing I can find anywhere suggests what happened to these reports, I cannot find them anywhere in the workspace, but without them my entire Jenkins pipeline would shatter.

_Originally posted by @anthonyemerson in https://github.com/sbt/sbt/discussions/6320_

**Comment by eed3si9n** [comment]
I should've locally tested. It works for me using `sbt new scala/scala-seed.g8` project, and `test`.

```
$ cat project/build.properties
sbt.version=1.4.7
$ ls target/test-reports
TEST-example.HelloSpec.xml
```

**Comment by anthonyemerson** [comment]
My apologies this can be closed, I had completely forgot that in SBT 1.2.8 both logs (foo.xml and TEST-foo.xml) were being generated so to cut back on excess noise we had a script cleaning up the TEST- log files.
